pandas/io/gbq.py

```diff
@@ -128,23 +130,44 @@ def test_google_api_imports(self):
         except ImportError as e:
             raise ImportError("Missing module required for Google BigQuery support: {0}".format(str(e)))
 
-    def get_credentials(self):
+    def authorize(self):
```

Comment:
no, you must have the credentials to pass in. pandas shouldn't be responsiblet for saving/retriveing these. ``.authorize`` should be almost a trivial call